### ISSUE TITLE
[NEW] Improve DynamicType support all over the app

### ISSUE THREAD

**Issue by rafaelks** [RocketChat/Rocket.Chat.iOS#2009]

**Comment by Sameesunkaria** [comment]
We should start working on it, as soon as we drop support for iOS 10.

**Comment by entonio** [comment]
Yes, please. Font size is just too small. Dynamic type or not, it's smaller than what you find on other apps, which aren't known for having large type either.
